mne/channels/layout.py

```diff
@@ -631,8 +631,16 @@ def _auto_topomap_coords(info, picks):
         locs3d = np.array([eeg_ch_locs[ch['ch_name']] for ch in chs])
 
     # Duplicate points cause all kinds of trouble during visualization
-    if np.min(pdist(locs3d)) < 1e-10:
-        raise ValueError('Electrode positions must be unique.')
+    dist = pdist(locs3d)
+    if np.min(dist) < 1e-10:
+        problematic_electrodes = [
+            info['ch_names'][elec_i]
+            for elec_i in np.unique(squareform(dist < 1e-10).nonzero()[0])
```

Comment:
I want the actual list of channels that overlap... I don't see how `(dist < 1e-10).any(axis=0)` gives me this list but I might just be too tired.